### DISCUSSION TITLE
Doubt Regarding Submission

### DISCUSSION THREAD

**Discussion by jaypatel31** [securestack-octernship/SecureStack-Octernship#9] [Q&A]
Hey there, I had completed assignment and I had applied through github classroom so, it created a private repo for me based on this one and as directed I created a new branch in my private repo. Do the changes and merge with main now in the feedback pr of my repo you can see the changes I made. So I have question do I need to to do anything else for submission or it is done ?

**Comment by shashwatw** [comment]
> Hey there, I had completed assignment and I had applied through github classroom so, it created a private repo for me based on this one and as directed I created a new branch in my private repo. Do the changes and merge with main now in the feedback pr of my repo you can see the changes I made. So I have question do I need to to do anything else for submission or it is done ?

What exactly u did to run this code in your local computer can u guide me too?

**Comment by jaypatel31** [comment]
I didn't run the code on my local computer, I just went through code https://github.com/SecureStackCo/app.cheapcryptobank.com.git present in this and used securestack to find the security vulnerability

**Comment by Exoutia** [comment]
> Hey there, I had completed assignment and I had applied through github classroom so, it created a private repo for me based on this one and as directed I created a new branch in my private repo. Do the changes and merge with main now in the feedback pr of my repo you can see the changes I made. So I have question do I need to to do anything else for submission or it is done ?

I also have the same question @6mile @ashwinexe @paulmccarty
